### DISCUSSION TITLE
auxiliary variable

### DISCUSSION THREAD

**Discussion by alimostafavi24** [idaholab/moose#23410] [Q&A General] [answered]
Hello everyone,

I am trying to define a new auxiliary variable which is w*grad(c). w an c are both primary variables.

I defined grad(c) components by VariableGradientComponent. My question is that what type of function I need to use in order to get a variable which is multiplication of grad(c) and w (w*grad(c)). I already tried ParsedFunction and failed.

Thanks,
Ali

**Comment by Minjiang-Zhu** [comment, marked as answer]
Try it in auxkernels.
